### ISSUE TITLE
Skip option

### ISSUE THREAD

**Issue by jplace** [coveo/fmt-maven-plugin#22]
It would nice if there was a way to skip the format check.

Something like:
```
mvn verify -Dfmt.skip
```

**Comment by rzwiefel** [comment]
This is not a permanent solution, but a viable workaround is to use 
`-DfilesNamePattern fileThatDoesntExist.java` in your maven command. 

credit to @mackncheesiest for finding this

**Comment by bryanschofield** [comment]
Nice!
```
$ mvn com.coveo:fmt-maven-plugin:2.2.0:format -Dfmt.skip=true
...
[INFO] ------------------------------------------------------------------------
[INFO] Building hl7-validator-resources 0.0.2-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO]
[INFO] --- fmt-maven-plugin:2.2.0:format (default-cli) @ hl7-validator-resources ---
[info] Skipping format check
...
```

**Comment by aaime** [comment]
This feature is available in 2.4.0, is this issue closeable then?
